Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:alpine
```

Comment:
This should perhaps specify the specific version, I believe 14 is the latest
`FROM node:14-alpine` I believe

Reply:
I think `node:alpine` is same `node:14-alpine` but yeah pinning a version is more stable.